### PR TITLE
all things ++

### PR DESCRIPTION
**Pull request by fbielejec**

### CODE REVIEW

File: devnet-azero/devnet-azero-compose.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+version: '3.9'
+
+services:
+  # key derived from "//0"
+  AlephBridgenode:
+    image: public.ecr.aws/p6e8q1z1/aleph-node:r-12.1
+    container_name: aleph_bridgenode
```
Comment (pmikolajczyk41) [inline devnet-azero/devnet-azero-compose.yml]: you might want to rename it (to exclude 'bridge' part); here and in the 26th line

File: README.md [hunk 2]
```diff
@@ -1,11 +1,66 @@
...
+Run devnet:
+
+```bash
+make devnet
+```
+
+Compile & deploy:
+
+```bash
+make compile
```
Comment (pmikolajczyk41) [inline README.md:54]: `sh: 1: typechain-compiler: not found`

Comment (pmikolajczyk41) [reply README.md:54]: maybe add an instruction `yarn install` above

**Review by pmikolajczyk41 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (fbielejec) [reply README.md:54]: Done

**Pull request merged**